openpype/hosts/flame/plugins/load/load_clip.py

```diff
@@ -38,8 +47,14 @@ def load(self, context, name, namespace, options):
         version_name = version.get("name", None)
         colorspace = self.get_colorspace(context)
 
+        # in case output is not in context replace key to representation
+        if not context["representation"]["context"].get("output"):
+            self.clip_name_template.replace("output", "representation")
```

Comment:
There two lines won't do anything.  Replace called on string won't change the source string but will return new.

Reply:
ah, yeah thanks